e2e/.env.example

```
@@ -0,0 +1,9 @@
+# GitHub App E2E Test Configuration
+# Copy this file to ../.env and fill in the values
+
+# Test GitHub account credentials (account must NOT have 2FA enabled)
+GITHUB_TEST_ACC_USERNAME=
+GITHUB_TEST_ACC_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_TEST_ACC_PASSWORD key should go before the GITHUB_TEST_ACC_USERNAME key

(UnorderedKey)